runelite-client/src/main/java/net/runelite/client/plugins/specialcounter/SpecialWeapon.java

```diff
@@ -54,7 +54,12 @@ public enum SpecialWeapon
 		(distance) -> 46 + distance * 10,
 		(c) -> 0
 	),
-	;
+	TONALZTICS_OF_RALOS(
+		"Tonalztics of Ralos",
+		new int[]{ItemID.TONALZTICS_OF_RALOS},
+		false,
+		(distance) -> 50, //The hitsplat is always applied 2t after spec regardless of distance
+		(c) -> 0);
```

Comment:
Does the special effect apply twice if each hitsplat hits? The wiki's information on this weapon doesn't seem to clarify this.

Reply:
Yes, it reduces by 10% of the target’s magic level per hitsplat that is > 0

Reply:
(yes, it does)